README.md

```diff
@@ -6,5 +6,7 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+Answer:  A web framework, Rails is a tool for building web apps written in Ruby. Architecture similar MVC.
+
```

Comment:
Though your diagram isn't wrong, it's not very specific. What distinguishes the different types of 'info' in your diagram?